bootstrap/sql/org.postgresql.Driver/v002__create_db_connection_info.sql

```diff
@@ -27,6 +27,18 @@ CREATE TABLE IF NOT EXISTS mlmodel_service_entity (
     UNIQUE (name)
 );
 
+CREATE TABLE IF NOT EXISTS objectstore_service_entity (
+    id VARCHAR(36) GENERATED ALWAYS AS (json ->> 'id') STORED NOT NULL,
+    name VARCHAR(256) GENERATED ALWAYS AS (json ->> 'name') STORED NOT NULL,
+    serviceType VARCHAR(256) GENERATED ALWAYS AS (json ->> 'serviceType') STORED NOT NULL,
+    json JSONB NOT NULL,
+    updatedAt BIGINT GENERATED ALWAYS AS ((json ->> 'updatedAt')::bigint) STORED NOT NULL,
+    updatedBy VARCHAR(256) GENERATED ALWAYS AS (json ->> 'updatedBy') STORED NOT NULL,
+    deleted BOOLEAN GENERATED ALWAYS AS ((json ->> 'deleted')::boolean) STORED,
+    PRIMARY KEY (id),
+    UNIQUE (name)
+);
+
```

Comment:
This is duplicated in 2 different migration script files. We should have only one, which should be in the latest file; otherwise, it will not be created when people migrate from previous versions.

Reply:
nice catch

Reply:
This is a bit embarrassing - I've mostly copy pasted things from the MLModel implementation. Will put these in the right place.

Reply:
Shouldn't these migration files be in the latest migration file. Right now it is in V001 and V002?